docs/man-openmpi/man3/MPIX_Query_rocm_support.3.rst

```diff
@@ -40,33 +40,55 @@ There is no C++ binding for this function.
 DESCRIPTION
 -----------
 
-This routine return 1 if MPI library is build with ROCm and runtime
-supports ROCm buffers. This routine must be called after MPI is
-initialized by a call to :ref:`MPI_Init` or :ref:`MPI_Init_thread`.
+This function is part of an :ref:`Open MPI extension
+<ompi-features-extentions-label>`; it is not part of standard MPI.  If
+Open MPI was built with the ROCm extenion, this function is declared
+in ``<mpi-ext.h>``.
+
```

Comment:
I don't think this is correct. This function always exists, whether Open MPI has been built with ROCm support or not. Otherwise it serves no purpose.

Reply:
I think the statement is technically correct: it says that the function will only be there if the ROCm extension was built.  It is built by default, but it *could* be disabled.

I'll soften the language.

Reply:
ok, you are right, I misread the line for 'ROCm support' vs. 'ROCm extension'